integration-cli/docker_api_containers_test.go

```diff
@@ -537,7 +538,7 @@ func TestPostContainerBindNormalVolume(t *testing.T) {
 	}
 
 	bindSpec := map[string][]string{"Binds": {fooDir + ":/foo"}}
-	_, err = sockRequest("POST", "/containers/two/start", bindSpec)
+	_, _, err = sockRequest("POST", "/containers/two/start", bindSpec)
```

Comment:
There are a lot of such places. Maybe makes sense to create issue about checking status there?